enterprise/views.py

```diff
@@ -58,11 +58,15 @@
     consent_necessary_for_course,
     enterprise_login_required,
     filter_audit_course_modes,
+    get_enterprise_customer_or_404,
 )
+from six.moves.urllib.parse import urljoin  # pylint: disable=import-error
 
 
 logger = getLogger(__name__)  # pylint: disable=invalid-name
 
+LMS_COURSE_URL = urljoin(settings.LMS_ROOT_URL, '/courses/{course_id}')
```

Comment:
Small nit; if there are no objections, can we change this out to the following?

```python
LMS_COURSE_URL = urljoin(settings.LMS_ROOT_URL, '/courses/{course_id}/courseware')
```

That'll take the learner to where they were in the course, rather than to the course homepage.

If anyone has any contrary thoughts, though, let's merge as-is, and have a discussion on the appropriate thing to do.